cmd/ssl-expiry-check/main.go

```diff
@@ -109,3 +212,17 @@ func reportKHFailure(errorMessage string) error {
 	log.Info("Successfully reported failure status to Kuberhealthy servers")
 	return err
 }
+
+func waitForNodeToJoin(ctx context.Context, client *kubernetes.Clientset, namespace *string) {
+	// Wait for node to be at least 2m old.
+	err := nodeCheck.WaitForNodeAge(ctx, client, *namespace, time.Minute*5)
+	if err != nil {
+		log.Errorln("Failed to check node age:", err.Error())
+	}
+
+	// Check if Kuberhealthy is reachable.
+	err = nodeCheck.WaitForKuberhealthy(ctx)
+	if err != nil {
+		log.Errorln("Failed to reach Kuberhealthy:", err.Error())
+	}
```

Comment:
this is repeated: https://github.com/Comcast/kuberhealthy/pull/658/files#diff-d7ac0c34789683f7b6664eede5c96959R108-R112

you already run this check `WaitForKuberhealthy` in your main func

Reply:
Removing from `main` and keeping in `waitForNodeToJoin`